### PR TITLE
feat: support ethereum on ledger

### PR DESCRIPTION
**Pull request by zctocm**
1. support Ethereum on ledger
  - `getAddress`
  - `signTransaction`
  - `signPersonalMessage`
  - `getAppConfiguration`
  - `getAddressList`
2. add signPersonalMessage API to Conflux on the ledger

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/723)

### CODE REVIEW

File: packages/ledger/conflux.js [hunk 1]
```diff
@@ -131,8 +147,9 @@ export default class Conflux {
       for (const index of indexArray) {
         const hdPath = `${HDPATH.CONFLUX}${index}`
         const {address} = await this.getAddress(hdPath)
+        const {hexAddress} = decode(address)
         addressArr.push({
-          address,
+          address: hexAddress,
```
Comment (yqrashawn) [inline packages/ledger/conflux.js:152]: will this broke the frontend code at https://github.com/Conflux-Chain/helios/blob/b1eda8c8712e394a2e999b9283c1db8b5a6ad982/packages/popup/src/pages/ImportHwAccount/index.js#L109

**Review by yqrashawn — COMMENTED** [review]
looks good with one minor issue

File: packages/ledger/conflux.js (hunk 1 above)
Comment (zctocm) [reply packages/ledger/conflux.js:152]: nope, the frontend code requires the hex-format address

**Review by yqrashawn — APPROVED** [review]
LGTM! :thumbsup:

**Pull request merged**